lib/diagnostic.js

```diff
@@ -32,7 +39,13 @@ db.dropDatabase();
 // Jon Arryn
 
 // your code begins here
-
+  db.houses.update({
+    name: "Stark"
+  },
+  {$set:{
+    "members": "[Ned Stark]"
```

Comment:
This will set a `members` key correctly, but the value will be the string `"[Ned Star]"` rather than an array.

`update()`'s first argument should be a query object.  The second argument should be an options object.  Here is a solution demonstrating the uses of `$set`, `$push`, and `$push` with `$each`.

```javascript
db.houses.update({
  name: 'House Stark'
}, {
  $set: {
    members: ['Ned Stark']
  }
});

db.houses.update({
  name: 'House Targaryen'
}, {
  $set: {
    members: ['Viserys Targaryen']
  }
});

db.houses.update({
  name: 'House Arryn'
}, {
  $set: {
    members: ['Jon Arryn']
  }
});

db.houses.update({
  name: 'House Stark'
}, {
  $push: {
    members: {
      $each: ['Arysa Stark', 'Sansa Stark']
    }
  }
});

db.houses.update({
  name: 'House Targaryen'
}, {
  $push: {
    members: 'Daenerys Targaryen'
  }
});
```